webviz_subsurface/containers/_summary.py

```diff
@@ -0,0 +1,670 @@
+            'y': ens_smry_data[ens_smry_data['REAL'] == real][vector],
+            'legendgroup': ens,
+            'name': ens,
+            'type': 'line',
+            'marker': {
+                'color': color
+            },
+            'showlegend': False
+        })
+
+    return ens_traces
+
+
+def single_trace(
+        ens_smry_data,
+        ens: str,
+        vector: str,
+        color: str):
+    """ function to create a single trace that shows up in the legend.
+
+    Args:
+        smry_data: pd.df = calculated summary data (fmu-ensemble)
+        ens: str = selected ensemble
+        color str = color
+    Returns:
+        trace-obj to be plotted in a plotly.fig
+    """
+
+    return {
+        'x': ens_smry_data[ens_smry_data['REAL']
+                           == ens_smry_data.REAL.unique()[0]]['DATE'],
+        'y': ens_smry_data[ens_smry_data['REAL']
+                           == ens_smry_data.REAL.unique()[0]][vector],
+        'legendgroup': ens,
+        'name': ens,
+        'type': 'markers',
+        'marker': {
+            'color': color
+        },
+        'showlegend': True
+    }
+
+
+def time_series_confidence_interval_traces(
+        vector_stats,
+        color_rgb: list,
+        legend_group: str
+        ):
+    """ function to create a convidence interval set of a selected ensemble.
+
+    Args:
+        smry_data: pd.df = calculated summary data (fmu-ensemble)
+        color (r: int, g:int, b:int) = rgb of color to be ploted
+        legend_group: str = selected ensemble
+    Returns:
+        list if trace-obj representing min, max, p10, p90 and mean
+    """
+
+    r, g, b = color_rgb
+
+    trace_maximum = go.Scatter(
+        name='maximum',
+        x=vector_stats['maximum'].index.tolist(),
+        y=vector_stats['maximum'].values,
+        mode='lines',
+        line=dict(width=0),
+        legendgroup=legend_group,
+        showlegend=False,
+    )
+
+    trace_p10 = go.Scatter(
+        name='p10',
+        x=vector_stats['p10'].index.tolist(),
+        y=vector_stats['p10'].values,
+        mode='lines',
+        fill='tonexty',
+        fillcolor='rgba({},{},{},{})'.format(r, g, b, 0.3),
+        line=dict(width=0),
+        legendgroup=legend_group,
+        showlegend=False,
+    )
+
+    trace_mean = go.Scatter(
+        name=legend_group,
+        x=vector_stats['mean'].index.tolist(),
+        y=vector_stats['mean'].values,
+        mode='lines',
+        fill='tonexty',
+        fillcolor='rgba({},{},{},{})'.format(r, g, b, 0.3),
+        line=dict(color='rgba({},{},{},{})'.format(r, g, b, 1)),
+        legendgroup=legend_group,
+        showlegend=True,
+    )
+
+    trace_p90 = go.Scatter(
+        name='p90',
+        x=vector_stats['p90'].index.tolist(),
+        y=vector_stats['p90'].values,
+        mode='lines',
+        fill='tonexty',
+        fillcolor='rgba({},{},{},{})'.format(r, g, b, 0.3),
+        line=dict(width=0),
```

Comment:
Use `{}` instead of where `dict` is used, at least for these simple ones.

E.g. `{'width': 0}` instead of `dict(width=0)`.

Reply:
Changed